java/core/src/java/org/apache/orc/impl/Dictionary.java

```diff
@@ -47,6 +47,15 @@
    */
```

Comment:
Can we please mention here that this method performs a copy from a byte array to Text?

Reply:
Donr.